bioarchaeology-of-the-near-east.csl

```diff
@@ -0,0 +1,594 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never">
```

Comment:
could you then add ```default-locale="en-US"``` to this line to make sure the style works in English for everyone.